docs/docs/extra-features.md

```diff
@@ -9,8 +9,8 @@ If you want to use a currently unsupported format feel free to submit a PR or co
 
 ## Source maps
 
-Source maps are enabled by default when running the `sku start` command.
-However, if you want to generate source maps when running `sku build`, you can do so by enabling [`sourceMapsProd`](./docs/configuration#sourcemapsprod).
+Source maps are enabled by default when running both `sku start` and `sku build`.
```

Comment:
Is there a reason they are enabled for `start`? Would assume this is dev and unminified, not requiring the source map?

Reply:
We still need sourcemaps in dev because otherwise logs/errors end up pointing directly to dev-server bundles which contain webpack stuff transpiled code, rather than source code, e.g. you see the error in `main.js` instead of the exact line in your `App.tsx`.